packages/eddsa-pcd/src/EdDSAPCD.ts

```diff
@@ -0,0 +1,147 @@
+import {
+  DisplayOptions,
+  PCD,
+  PCDPackage,
+  SerializedPCD,
+  StringArgument,
+  StringArrayArgument
+} from "@pcd/pcd-types";
+import { buildEddsa, buildPoseidon } from "circomlibjs";
+import { v4 as uuid } from "uuid";
+import { EdDSACardBody } from "./CardBody";
+
+// eslint-disable-next-line @typescript-eslint/triple-slash-reference
+/// <reference path="util/declarations/circomlibjs.d.ts" />
+
+export const EdDSAPCDTypeName = "eddsa-pcd";
+
+export interface EdDSAPCDArgs {
+  privateKey: StringArgument;
+  message: StringArrayArgument;
+  id: StringArgument;
+}
+
+export interface EdDSAPCDClaim {
+  message: bigint[];
```

Comment:
`publicKey` should be in claim, not proof. see [here](https://0xparc.notion.site/2023-03-29-Brian-Proof-vs-Claim-8f4966c4dbd345bfa02e28422a620ba3?pvs=4) for guidelines on claim vs proof

Reply:
fwiw I copied this from RSAPCD. I'm not exactly sure why this is different, but I infer from the linked document that it has something to do with how application developers might interact with the PCD.